dbt/adapters/athena/impl.py

```diff
@@ -1013,6 +1014,29 @@ def persist_docs_to_glue(
                     # Save column description from dbt
                     col_obj["Comment"] = clean_col_comment
 
+                    # Get dbt model column meta if available
+                    col_meta: Dict[str, Any] = model["columns"][col_name].get("meta", {})
+                    # Add empty Parameters dictionary if not present
+                    if len(col_meta) > 0 and "Parameters" not in col_obj.keys():
+                        col_obj["Parameters"] = {}
+                    # Prepare meta values for column properties and check if update is required
+                    for meta_key, meta_value_raw in col_meta.items():
+                        if is_valid_table_parameter_key(meta_key):
+                            meta_value = stringify_table_parameter_value(meta_value_raw)
+                            if meta_value is not None:
+                                # Check if meta value is already attached to Glue column
+                                col_current_meta_value: Optional[str] = col_obj["Parameters"].get(meta_key)
```

Comment:
If you remove the Parameters check as suggested [here](https://github.com/dbt-athena/dbt-athena/pull/644/files#r1595563563), you might need to readapt this, because Parameters could be empty, therefore `col_obj.get("Parameters", {}) .get(meta_key)` should work.